applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1201,12 +1200,16 @@ static void disconnect(const void *subscriber_id, uint8_t report_id)
 	update_output_report_state();
 }
 
-static void connect_subscriber(const void *subscriber_id, bool is_usb, uint8_t report_max)
+static void connect_subscriber(const void *subscriber_id, uint8_t priority,
+			       uint8_t pipeline_size, uint8_t report_max)
 {
+	__ASSERT_NO_MSG(priority > 0);
```

Comment:
```suggestion
	__ASSERT_NO_MSG(priority > 0);
	__ASSERT_NO_MSG(pipeline_size > 0);
	__ASSERT_NO_MSG(report_max > 0);
```
To make the check complete